client/src/resources/style-constants.js

```diff
@@ -32,6 +36,8 @@ export const fontSize64 = "3rem";
 /** Border Radiuses */
 export const borderRadius = "12px";
 
+/* Other */
+export const appBarHeight = "96px";
```

Comment:
I like this notion of adding common-component stylings to this file; good thinking.